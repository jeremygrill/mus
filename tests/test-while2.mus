Int main(){
	
	Int i;
	Note n;
	Seq s;

	i = 0;

	while(i < 24){
		n = (60 + i, 10|10);
		s = s + [n];
		printn(n);
		i = i +1;
	}

	prints(s);

	return 0;

}
