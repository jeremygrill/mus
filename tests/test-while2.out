(60,10|10)
(61,10|10)
(62,10|10)
(63,10|10)
(64,10|10)
(65,10|10)
(66,10|10)
(67,10|10)
(68,10|10)
(69,10|10)
(70,10|10)
(71,10|10)
(72,10|10)
(73,10|10)
(74,10|10)
(75,10|10)
(76,10|10)
(77,10|10)
(78,10|10)
(79,10|10)
(80,10|10)
(81,10|10)
(82,10|10)
(83,10|10)
$([(60,10|10)][(61,10|10)][(62,10|10)][(63,10|10)][(64,10|10)][(65,10|10)][(66,10|10)][(67,10|10)][(68,10|10)][(69,10|10)][(70,10|10)][(71,10|10)][(72,10|10)][(73,10|10)][(74,10|10)][(75,10|10)][(76,10|10)][(77,10|10)][(78,10|10)][(79,10|10)][(80,10|10)][(81,10|10)][(82,10|10)][(83,10|10)])
